transcom/mymove
# B-21460-INT Mobile Home and Boat shipments now have 2nd/3rd Address

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1015581&RoomContext=TeamRoom%3A848240&concept=TeamRoom)

## Summary

Fixed issue with Prime Sim created shipments not having 2nd/3rd addresses

### How to test

1. Access the office app
2. Login as a Prime Sim
3. Open a move
4. Create a new Boat/MH shipment with 2nd/3rd addresses
5. login as TOO/TIO/SC and open that move
6. Verify shipment has 2nd/3rd addresses and H&AL shows 2nd/3rd addresses for that shipment
